### DISCUSSION TITLE
Repo is hackotberfest excluded

### DISCUSSION THREAD

**Discussion by Sohamshah03** [abhisek247767/AI-ML-DL-Hacktoberfest2024-WB#123] [Q&A] [answered]
@abhisek247767 please look into it ,the repo is hackotberfest excluded.

**Comment by abhisek247767** [comment, marked as answer]
Hi @Sohamshah03 , we're currently investigating why our repository was excluded from Hacktoberfest. Initially, it was included for the first 17 days, and we're doing our best to restore its eligibility. We apologize for the confusion and will keep you updated.
